packages/core/src/QueryBuilder/Rule/Value/Value.tsx

```diff
@@ -37,7 +37,7 @@ export const Value = ({
     useQueryBuilderContext();
 
   const attrType =
-    attribute && attributes ? attributes[attribute].type : undefined;
+    attribute && attributes ? attributes[attribute]?.type : undefined;
```

Comment:
According to our types, `attribute` is always a string. Even if it were `undefined`, `({})[undefined]` wouldn't blow up - it just returns `undefined`.
```suggestion
    attributes?.[attribute]?.type;
```

I'm just suggesting this because (besides being simpler) the [operator precedence of the ternary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#table) isn't obvious to be, and I had to check it 😅 
```tsx
// which one is it?
attribute && (attributes ? attributes[attribute]?.type : undefined);
(attribute && attributes) ? attributes[attribute]?.type : undefined;
```

Reply:
I'll update this